README.md

```diff
@@ -31,14 +31,16 @@ positional arguments:
   filename
 
 options:
-  -h, --help            show this help message and exit
-  -e, --exif            print photo exif data on the border
-  -p, --palette         Add colour palette to the photo border
-  -t, --border_type     Border Type: p for polaroid, s for small, m for medium, l for large, i for instagram (default: s)
-  -f, -font             Font Typeface to use
-  -fb, --font bold      Bold Font Typeface to use
-  --include             File patterns to include (default: *.jpg *.jpeg *.png, *.JPG, *.JPEG, *.PNG)
-  --exclude             File patterns to exclude (default: *_border*)
+  -h, --help              Show this help message and exit
+  -e, --exif              Print photo exif data on the border
+  -p, --palette           Add colour palette to the photo border
+  -t, --border_type       Border Type: p for polaroid, s for small, m for medium, l for large, i for instagram (default: s)
+  -f, --font              Font Typeface to use (default: Roboto-Regular.ttf)
+  -fv, --fontvariant      Font style variant to use (default: 0)
```

Comment:
Do we need to explain how to get the font varient? Is it an integer or string?

Reply:
It's an int, but I think that is explained in the argparser help as well as the default: 0 in the readme. 

I'm actually not sure of the best way to get the index int before running which is one of the reasons I added all the variants to the error output, eg:

```
Roboto-Regular.ttf does not contain a variant with index: 1. Available variants: [(0, {'family': 'Roboto', 'style': 'Regular'})]"]
```

I think that's probably ok for now. I'll merge this in. 

Thanks for review 👍